client/blocks/importer/wordpress/upgrade-plan/style.scss

```
@@ -122,6 +122,49 @@ $business-plan-color: #7f54b3;
 		}
 	}
 
+	.import__upgrade-plan-price-group {
+		justify-content: flex-end;
+		display: flex;
+		flex-direction: row-reverse;
+		align-items: baseline;
+		gap: 4px;
+
+		.plan-price.is-original {
+			font-size: 1.25rem;
+
+			&::before {
+				border-top: 2px solid var(--color-neutral-light);
+				transform: rotate(0deg);
+			}
+
+			.plan-price__currency-symbol {
+				font-size: 0.75rem;
+				vertical-align: text-top;
+			}
+
+			.plan-price__integer {
+				font-size: 1.25rem;
+			}
+		}
+	}
+
+	.import__upgrade-plan-price-badge {
+		text-align: center;
+		font-size: 0.75rem;
+		line-height: 1.5;
+		height: auto;
+		border-radius: 4px;
+		display: inline-block;
+		width: fit-content;
+		letter-spacing: 0.2px;
+		font-weight: 500;
+		padding: 0 12px;
+		background-color: #e6d1ff;
+		color: #350e63;
+		text-transform: none;
+		margin-bottom: 0.5rem;
+	}
```

It feels like we're adding too much CSS here. Based on my testing, I think we'll get very similar UX if we remove the styles in my suggested edit, which also means we can pick up general changes to the Badge component and be more in sync with changes across the platform. To be clear: I am erring on the side of less long-term maintenance and more general visual consistency over pixel-perfect matching the designs. It may be worth getting input/review from the designer before making drastic changes.

```suggestion
	.import__upgrade-plan-price-badge {
		font-size: 0.75rem;
		border-radius: 4px;
		width: fit-content;
		font-weight: 500;
		background-color: #e6d1ff;
		color: #350e63;
		margin-bottom: 0.5rem;
	}
```

Also, I see that the colours don't match any colours in our default set of colour variables. I think that's non-blocking in terms of getting this shipped, _but_ I think we should double-check with the designer about possibly using colours from our palette so they are more future-proof, and revisit that pretty soon (mostly so we don't forget).